Config.in

```diff
@@ -6,4 +6,6 @@ source "$BR2_EXTERNAL_FTCOMMUNITY_TXT_PATH/package/python-libroboint/Config.in"
 source "$BR2_EXTERNAL_FTCOMMUNITY_TXT_PATH/package/python-robolt/Config.in"
 source "$BR2_EXTERNAL_FTCOMMUNITY_TXT_PATH/package/python-semantic-version/Config.in"
 source "$BR2_EXTERNAL_FTCOMMUNITY_TXT_PATH/package/python-zbarlight/Config.in"
+source "$BR2_EXTERNAL_FTCOMMUNITY_TXT_PATH/package/touchui/Config.in"
+source "$BR2_EXTERNAL_FTCOMMUNITY_TXT_PATH/package/touchui-system-apps/Config.in"
```

Comment:
I'm not sure if it makes sense to pull the system apps out of ftcommunity-TXT. After all, most of them are pretty much tied to the CFW and not usable anywhere else.